modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -106,6 +106,14 @@ The Ingress Operator also converts the TLS `1.0` of an `Old` or `Custom` profile
 Ciphers and the minimum TLS version of the configured security profile are reflected in the `TLSProfile` status.
 ====
 
+|`clientTLS`
+|`clientTLS` authenticates client access to the cluster and services; as a result, mutual TLS authentication is enabled. If not set, then client TLS is not enabled.
+
+`clientTLS` has the required subfields, `spec.clientTLS.clientCertificatePolicy` and `spec.clientTLS.ClientCA`.
+
+The `ClientCertificatePolicy` subfield accepts one of the two values: `Required` or `Optional`. The `ClientCA` subfield specifies a ConfigMap that is in the openshift-config namespace. The ConfigMap should contain a CA certificate bundle.
```

Comment:
For the field names like `clientCertificatePolicy`, `clientCA`, and `allowedSubjectPatterns`, please revise to match the same initial lowercase first letter that that API uses, like you have for `clientTLS`.

In lines 114 to 116, please replace ConfigMap with "config map" according to the guidelines for [general references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-general-references).

I wish that I knew more about the `allowedSubjectPatterns`.  The "if left empty" makes me think it would be better as "if not specified," but I regrettably do not know the behavior.